Sources/AWSLambdaRuntime/FoundationSupport/Lambda+JSON.swift

```diff
@@ -85,10 +87,12 @@ extension LambdaCodableAdapter {
 extension LambdaRuntime {
     /// Initialize an instance with a `LambdaHandler` defined in the form of a closure **with a non-`Void` return type**.
     /// - Parameters:
+    ///   - logger: The logger to use for the runtime. Defaults to a logger with label "LambdaRuntime".
     ///   - decoder: The decoder object that will be used to decode the incoming `ByteBuffer` event into the generic `Event` type. `JSONDecoder()` used as default.
     ///   - encoder: The encoder object that will be used to encode the generic `Output` into a `ByteBuffer`. `JSONEncoder()` used as default.
     ///   - body: The handler in the form of a closure.
     public convenience init<Event: Decodable, Output>(
+        logger: Logger = Logger(label: "LambdaRuntime"),
```

Comment:
It is common for the logger parameter to be the last parameter (or at least the last before any optional parameters and closures).